### PR TITLE
Adding Polis

### PR DESCRIPTION
**Pull request by zeusthealmighty**

### CODE REVIEW

File: slip-0044.md [hunk 1]
```diff
@@ -297,6 +298,7 @@ index | hexa       | symbol | coin
 5718350 | 0x8057414e | WAN    | [Wanchain](https://wanchain.org/)
 5741564 | 0x80579bfc | WAVES  | [Waves](https://wavesplatform.com/)
 
+
```
Comment (prusnak) [inline slip-0044.md]: Please remove the extra line.

Comment (zeusthealmighty) [reply slip-0044.md]: Done!

**Comment by zeusthealmighty** [comment]
Done!

2018-05-14 9:15 GMT-05:00 Pavol Rusnak <notifications@github.com>:

**Comment by zeusthealmighty** [comment]
So... ?

**Pull request merged**